config/v1/feature_gates.go

```diff
@@ -595,4 +595,11 @@ var (
 						productScope(ocpSpecific).
 						enableIn(TechPreviewNoUpgrade).
 						mustRegister()
+
+	FeatureGateVSphereDriverConfiguration = newFeatureGate("VSphereDriverConfiguration").
+						reportProblemsToJiraComponent("Storage / Kubernetes External Components").
+						contactPerson("rbednar").
+						productScope(ocpSpecific).
+						enableIn(Default, TechPreviewNoUpgrade).
```

Comment:
TechPreview first please. Demonstrate stability (either via automation or QE sign off) and we can promote in 4.16.

Reply:
Before becoming accessible-by-default, we must have evidence of completeness and reliability.  The preferred mechanism is to run automated CI tests in our existing TechPreview jobs and see 95+% pass rates over at least 14 runs.  The alternative mechanism is to have QE sign off on a promotion PR.  That can all happen in a single release.  For instance,

1. in 4.16
2. merge as techpreview
3. add functionality and tests
4. after there are 14 runs, open PR promoting to GA
5. CI automatically checks for the associated tests (see example failure [here](https://prow.ci.openshift.org/view/gs/test-platform-results/pr-logs/pull/openshift_api/1851/pull-ci-openshift-api-master-verify/1778771838210412544))
6. PR is merged on green OR QE signs off on the PR and we override the automated stability check (we track data on this now in 4.16)
7. we ship 4.16


So this doesn't delay availability in 4.16 assuming the feature is tested and is as stable as expected

Reply:
@deads2k Removed Default cluster profile for this feature, can we merge this now as TechPreview?